c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -16,37 +26,78 @@ Version 1.6
 2. Removes the deprecated files:
 
     git rm -r {{package}}/CONST_migration
-
-3. In the {{package}}/__init__.py file do the following changes:
+    git rm -r {{package}}/static/build
+    git rm config.yaml
+    git rm bootstrep.py
+    git rm -r *buildout*
+    git rm development.ini.in production.ini.in
+    git rm versions.cfg
+
+3. In this version we use a new build system based on make that will change
+   many of your habits, than read, than read:
+   http://docs.camptocamp.net/c2cgeoportal/1.6/integrator/make.html
+
+   Main things to do:
+   - Move your old `config.yaml` and your `vars` section of the old
+     `buildout.cfg` file to the new `{{package}}.yaml` file.
+   - Move your special task from your old `buildout.cfg` file to your new
```

Comment:
Should parts for buildout and rule for make: https://www.gnu.org/software/make/manual/make.html#Rule-Example
=> ?
Move your special buildout parts from your old `buildout.cfg` file as a make rule in your new `{{package}}.yaml` file.